src/cascadia/TerminalConnection/Resources/en-US/Resources.resw

```diff
@@ -148,6 +148,10 @@
     <value>You have not set up your cloud shell account yet. Please go to https://shell.azure.com to set it up.</value>
     <comment>{Locked="https://shell.azure.com"} This URL should not be localized. Everything else should.</comment>
   </data>
+  <data name="AzureInvalidUserSettings" xml:space="preserve">
+    <value>Your cloud shell account misses some mandatory settings. Please go to https://shell.azure.com to set it up.</value>

```

Comment:
```suggestion
    <value>Your cloud shell account does not have a default shell configured. Please go to https://shell.azure.com to set it up.</value>
```

Perhaps? We'd need to workshop the text if we go with the graceful fallback solution